Gruntfile.js

```diff
@@ -141,6 +142,31 @@ module.exports = function(grunt) {
 				}]
 			}
 		},
+		postcss: {
+			defaults: {
+				options: {
+					map: {
+						inline: false, // Save all sourcemaps as separate files.
+					},
+					processors: [
+						autoprefixer, // add vendor prefixes.
+					],
+				},
+				src: ['ie.css', 'ie.min.css', 'style.css', 'style.min.css'],
```

Comment:
Do we actually want to run this on IE stylesheets? They're only loaded on older IE - the ones that support conditional comments.